cfgov/unprocessed/css/organisms/form.less

```diff
@@ -51,8 +51,13 @@
     margin-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em);
 
     .respond-to-max(@bp-sm-max, {
-        .btn {
+        .a-btn {
             width: 100%;
```

Comment:
So these aren't totally equivalent. `a-btn__full-on-xs` turns into a full width at a smaller size than this one does. This is kinda a weird one because it's used in the submit-button atom, which is *only* used on the conference registration form that appears as a component in wagtail. Could probably use `a-btn__full-on-xs`, but it does change the appearance so probably better to do it in another PR.